mbed_flasher/flashers/FlasherMbed.py

```diff
@@ -178,10 +179,11 @@ def try_drag_and_drop_flash(self, source, target, no_reset):
             return self.verify_flash_success(
                 target, MbedCommon.get_binary_destination(target["mount_point"], source))
         # In python3 IOError is just an alias for OSError
-        except (OSError, IOError) as err:
-            msg = "Write failed due to OSError: {}".format(err)
-            self.logger.error(msg)
-            raise FlashError(message=msg, return_code=EXIT_CODE_OS_ERROR)
+        except (OSError, IOError):
+            msg = "File copy failed due to OSError"
```

Comment:
could be also IOError since both errors are catch in same place. Maybe better to split to be able to track which one happens...

Reply:
In python3 IOError is removed, it is just an alias to OSError. I don't see any point to separately catch them.

Reply:
python2 there is different explanations:

https://docs.python.org/2/library/exceptions.html#exceptions.IOError
> Raised when an I/O operation fails for an I/O-related reason

https://docs.python.org/2/library/exceptions.html#exceptions.OSError
> It is raised when a function returns a system-related error

as long as we uses heavily python2 this matters.. (see IOTSYSTOOL-531, IOTSYSTOOL-770)

Reply:
Seems that in practise it is not so black-and-white as mentioned in documentation. Let's continue with this approach...